cime_config/machines/template.case.run.sh

```diff
@@ -43,6 +44,16 @@ export OMP_NUM_THREADS=$(./xmlquery THREAD_COUNT --value)
 cd $(./xmlquery RUNDIR --value)
 {{ mpirun }}
 
+# get timing
+python3 -c '
+import os
+from CIME.case import Case
+from CIME.get_timing import get_timing
+
+with Case(os.environ["CASEROOT"], read_only=True) as case:
+    get_timing(case, os.environ["LID"])
```

Comment:
just curios: since this script has the variable $lid, why use a call to os.environ?

Reply:
In bash, the shell does not try to evaluate syntax inside single-quoted strings. This is what I want since I don't want bash getting confused by python syntax. This means I can't just drop in `$lid` and have it work. That said, it's not hard to terminate the single quoted string and concatenate the variables, so I will do that.